wowchemy-cms/templates/single.html

```diff
@@ -11,5 +11,26 @@
 <body>
 <!-- Include the latest Netlify CMS v2.x.x script that builds the admin panel. -->
 <script src="https://cdn.jsdelivr.net/npm/netlify-cms@2/dist/netlify-cms.min.js"></script>
+{{ $shortcodes := resources.Get "js/shortcodes-cms.js" }}
+<script src="{{ $shortcodes.RelPermalink }}"></script>
+{{ $post_template := resources.Get "js/cms-preview-templates/post.js" }}
+<script src="{{ $post_template.RelPermalink }}"></script>
+{{ $custom_preview_css := resources.Get "css/custom-preview.css" }}
+<script>
+  // Register any CSS file on the home page as a preview style
+  fetch("/")
+  .then(response => response.text())
+  .then(html => {
+    const f = document.createElement("html");
+    f.innerHTML = html;
+    Array.from(f.getElementsByTagName("link")).forEach(tag => {
+      if (tag.rel == "stylesheet") {
+        CMS.registerPreviewStyle(tag.href);
+      }
+    });
+    CMS.registerPreviewStyle("{{ $custom_preview_css.RelPermalink }}"); // custom preview CSS
```

Comment:
Loads `custom-preview.css` for additional user-customizable styling.

Reply:
We shouldn't be adding any executable Go code inside JS code - not only is it bad practice, but also how are other contributors going to know that you've been writing Go inside Javascript? Please can you address those instances?  Thanks

Reply:
@gcushen How should this be addressed? Files such as https://github.com/wowchemy/wowchemy-hugo-themes/blob/main/wowchemy/layouts/partials/comments/disqus.html similarly add executable Go code inside JS code.

Reply:
Much of the code originates from 5+ years ago, before Hugo introduced better functionality for handling this scenario.

Since Hugo 0.78, the best practice is to use the new `js.Build` capability to import Hugo parameters into JS. For example:

```
{{ $js := resources.Get "js/main.js" | js.Build (dict "params" (dict "key" "value")) }}
```

And then in your JS file:

```
import * as params from '@params';
```

It's all documented in the Hugo Docs and Hugo Forums.

We also already use the js.Build feature in a number of places in the code if you'd like to see some more examples.